Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -470,6 +470,40 @@ final class UIViewExtensionsTests: XCTestCase {
         XCTAssertEqual(buttonSubview.ancestorView(withClass: UITableView.self), tableView)
     }
 
+    func testFindConstraint() {
+        let view = UIView()
+        let container = UIView()
+        container.addSubview(view)
+        view.widthAnchor.constraint(equalToConstant: 1).isActive = true
```

Comment:
Can you add a test case for a view that is not on the container hierarchy... a view that It is not a subview of container?

Reply:
sure